### ISSUE TITLE
11.0 mail thread widget not showing up inside notebook page

### ISSUE THREAD

**Issue by elgow** [odoo/odoo#20032]
Impacted versions: 11

Steps to reproduce: Define a view with an arch that has the following structure
```
<form>
  <notebook>
     <page>
         ...
               <div class="oe_chatter">
                     <field name="message_follower_ids" widget="mail_followers"/>
                      <field name="message_ids" widget="mail_thread"/>
                </div>
     </page>
    <page>
        .....
    </page>
   </notebook>
</form>
```
Current behavior:  The oe_chatter div will not be rendered on the page.  If the oe_chatter div is moved outside of the notebook then it does show up normally. 

Expected behavior:  In Odoo 10.0 and 9.0 the oe_chatter div would be rendered in a notebook page, allowing the notebook pages to get more value from the screen real estate.  

Video/Screenshot link (optional):

**Comment by elgow** [comment]
I should note that one of my notebook tabs has a document editor in it and showing the mail thread on that tab is quite damaging to the UI. The pre-11.0 behavior had great value.

**Comment by elgow** [comment]
@Yenthe666 Could you please tag this as 11.0 since it represents a breaking change from 9.0 and 10.0 for anyone trying to manage screen real estate by not having the very large chatter display appear on every tab in a notebook.

**Comment by Yenthe666** [comment]
Done! I believe @sle-odoo can give feedback on this matter.

**Comment by elgow** [comment]
Thank you for your help. I've tried various HTML arrangements to see if I can make it work, but no luck so far.

**Comment by sle-odoo** [comment]
@ged-odoo

**Comment by mge-odoo** [comment]
Hi @elgow 

Thanks for reporting this issue. It has been solved in 11.0

Note that the chatter style is only meant to be used as a direct child of `<form>` so it might be a bit ugly if displayed inside a tab for example. But if you want to do so, you will probably fix this layout issue in a few css rules in your custom module.

Regards,

Martin

**Comment by elgow** [comment]
Hi @mge-odoo 

Many thanks to you, Martin, for fixing this so quickly. Selective display of the chatter really helps to de-clutter my UI tabs. 

  Ed
